include/z64player.h

```diff
@@ -726,7 +726,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE1_20 (1 << 20)
 #define PLAYER_STATE1_21 (1 << 21)
 #define PLAYER_STATE1_SHIELDING (1 << 22) // Shielding in any form (regular, hylian shield as child, "shielding" with a two handed sword, etc.)
-#define PLAYER_STATE1_23 (1 << 23)
+#define PLAYER_STATE1_ON_HORSE (1 << 23)
```

Comment:
`// Currently on a horse. This includes getting on and getting off of the horse.`